src/baldr/directededge.cc

```diff
@@ -203,6 +203,16 @@ void DirectedEdge::set_bridge(const bool bridge) {
   bridge_ = bridge;
 }
 
+// Sets the hov type.
+void DirectedEdge::set_hov_type(const HOVEdgeType hov_type) {
+  hov_type_ = static_cast<uint32_t>(hov_type);
+}
+
+// Is this edge strictly hov? (if so, it could be hov2 or hov3, check the hov_type_)
```

Comment:
frankly i wonder if it would matter if we didnt just change the semantics of the bit to not include autoaccess. how many people could have possibly been actually using the HOV costing and how many of those wont update their data and code soon after we release this? famous last words i know, but this is the first time i feel pretty confident that HOV was an under-used thing